grails-app/services/org/pih/warehouse/core/OrganizationService.groovy

```diff
@@ -45,7 +45,18 @@ class OrganizationService {
     }
 
     Organization findOrCreateOrganization(String name, String code, List<RoleType> roleTypes) {
-        Organization organization = Organization.findByCodeOrName(code, name)
+        Organization organization = org.pih.warehouse.core.Organization.find {
```

Comment:
This doesn't feel right. I think I'd rather throw an exception if the code or name are empty than change all finders in the system to perform this type of search. Note that what we come up with here might be used throughout the system since there are a lot of places where we want to do a lookup by code or name.

If we clean up the OBNAV data and prevent users from entering blank values into code and name, then that should probably be enough.